packages/code-du-travail-api/src/server/routes/search/search.getSavedResult.js

```diff
@@ -48,7 +48,7 @@ async function _getPrequalified() {
 
 const getPrequalified = memoizee(_getPrequalified, {
   promise: true,
-  maxAge: 1000 * 5,
+  maxAge: 1000 * 5 * 60,
```

Comment:
C'est à peu près 3 jours ?

Reply:
5min si j'ai pas fait nimp https://github.com/medikoo/memoizee#expire-cache-after-given-period-of-time

Reply:
yep c'est des ms, my bad